backend/burning-okr/burning-okr-app/src/main/resources/db/migration/sqlserver/V18__AddPhotoCoumnToOkrUnit.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE okr_unit
+    ADD photo VARCHAR(255);
```

Comment:
Currently we only allow Images with a size less than 2MB. Lets assume we keep it that way for now:

According to [Wikipedia ](https://en.wikipedia.org/wiki/Base64#:~:text=bytes%20%3D%20(string_length(encoded_string)%20%E2%88%92%20814)%20/%201.37) the size of an image in bytes can be estimated by
bytes = (string_length(encoded_string) − 814) / 1.37

So we get
string_length(encoded_string) = 1.37 * bytes + 814

In our case byte equal 2MB = 2*1024*1024, resulting in
string_length(encoded_string) = 2.873.913 

Since in UTF-8 each character is represented by a single byte we need to store a string with a byte size of 
2.873.913 B = 2.75 MB

So for now we could just user Varchar(3.000.000) as an upper bound.

ALTERNATIVELY we could simply reduce the allowed image size to a fraction of 2MB since the okr-unit profile images are tiny anyway